pkg/execution/svccatadmission/types.go

```diff
@@ -12,6 +14,7 @@ import (
 const (
 	serviceInstance = "serviceinstances"
 	serviceBinding  = "servicebindings"
+	ingress         = "ingresses"
```

Comment:
These constants can go into `pkg/k8s` package.

Reply:
fixed. Note that this refactoring added changes to files not related to this PR